src/coreclr/tests/src/Regressions/coreclr/GitHub_34094/test34094.cs

```diff
@@ -0,0 +1,557 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Are these auto-generated with T4? If so, can you attach the *.tt file?

Reply:
This was just hand written since its a regression test, there is no template.